README.md

```diff
@@ -1,20 +1,43 @@
-![](assets/banner.png)
+# Delta
 
-Delta is a **blazing fast API server** built with Rust for the REVOLT platform.
+## Description
+
+Delta is a blazing fast API server built with Rust for Revolt.
+
+**Features:**
 
-Features:
 - Robust and efficient API routes for running a chat platform.
 - Distributed notification system, allowing any node to be seamlessly connected.
 - Simple deployment, based mostly on pure Rust code and libraries.
 - Hooks up to a MongoDB deployment, provide URI and no extra work needed.
 
-## Docker Helper Scripts
+## Stack
+
+- [Rocket](https://rocket.rs/) (REST)
+- [Async Tungstenite](https://github.com/sdroege/async-tungstenite) (WebSockets)
+- [MongoDB](https://mongodb.com/)
+
+## Resources
+
+### Revolt
+
+- [Revolt Project Board](https://github.com/revoltchat/revolt/discussions) (Submit feature requests here)
+- [Revolt Testers Server](https://app.revolt.chat/invite/Testers)
+- [Contribution Guide](https://developers.revolt.chat/contributing)
+
+## CLI Commands
+
+### Docker Helper Scripts
+
+If you have Docker installed, you can use the helper scripts to deploy Delta in your development environment.
+
+| Command        | Description                |
+| -------------- | -------------------------- |
+| `./build.sh`   | Build Docker Image.        |
+| `./run.sh`     | Run Docker container.      |
+| `./monitor.sh` | View container logs.       |
+| `./remove.sh`  | Kill and remove container. |
```

Comment:
These are no longer relevant, could mention the following though:
- `./publish.sh` used by me to publish Docker images
- `./set_version.sh` used by me to update version, not intended for PR use
- `cargo build` to build
- `cargo run` to run
- `cargo fmt` to format code (not for PRs either, since there might be code that hasn't been formatted yet)